pymc3/math.py

```diff
@@ -474,3 +474,26 @@ def block_diagonal(matrices, sparse=False, format="csr"):
     if len(matrices) == 1:  # graph optimization
         return matrices[0]
     return BlockDiagonalMatrix(sparse=sparse, format=format)(*matrices)
+
+
+def _linear_training_data(size=100):
```

Comment:
No need to have size as an argument, this function should only be used for that example. Also I would mention in the docstring that this is exactly the purpose of the function.